metricbeat/include/fields/fields.go

```diff
@@ -24,13 +24,13 @@ import (
 )
 
 func init() {
-	if err := asset.SetFields("metricbeat", "../libbeat/fields.yml", asset.LibbeatFieldsPri, AssetLibbeatFieldsYml); err != nil {
+	if err := asset.SetFields("metricbeat", "fields.yml", asset.BeatFieldsPri, AssetFieldsYml); err != nil {
```

Comment:
I'm not sure if this is correct. Previously it was generated based of the libbeat fields.yml which only contains the libbeat fields. Now it seems to be changed to the metricbeat fields.yml which likely also contains more fields from metricbeat itself. Do we have duplicated fields now or are by chance missing some?

Reply:
I checked for duplicates and we didn't have any. I will try to understand why. 
But my first approach was to use `../libbeat/fields.yml`  for that fields.go.  It should be the best solution, but there as some other fields missing which are the [common fields](https://github.com/elastic/beats/blob/master/metricbeat/_meta/fields.common.yml).
We would  need a fields.go for them also and then it starts getting more complicated.